presto-hive/src/main/java/com/facebook/presto/hive/HiveClientConfig.java

```diff
@@ -80,6 +80,8 @@
     private DataSize s3MultipartMinFileSize = new DataSize(16, MEGABYTE);
     private DataSize s3MultipartMinPartSize = new DataSize(5, MEGABYTE);
 
+    private boolean parquetColumnIndexAccess = false;
```

Comment:
The default should be to access columns via index since that is how Hive works.  For those that want the other behavior they should turn this off.